spec/std/time/time_spec.cr

```diff
@@ -1027,6 +1027,35 @@ describe Time do
     end
   end
 
+  describe Time::DayOfWeek do
+    it "#value" do
```

Comment:
I believe `describe` would fit better to the method definition.

Reply:
This is a single test case including multiple expectations. Do you think each line should be it's own test case?

Reply:
No. It makes sense to keep all of them in a single test case, I was just wondering `describe` fits better with the methods names and the`it` could be a description of the expectation, that would be a "documentation for free".

i.e: 

```crystal
describe "#value" do
  it "returns the number representation of the weekday" do
    #...
  end
end
```

what do you think?

Reply:
I generally appreciate expressiveness, but in this case I think it's just needlessly overly verbose. What's the benefit of repeating the description of the method which is already documented with its definition? If there are multiple test cases, yes, then it makes sense to describe them more explicitly. But this is just a simple spec that ensures `#value` behaves like it is supposed to.